crates/analytics/src/payments/distribution.rs

```diff
@@ -24,6 +24,8 @@ pub struct PaymentDistributionRow {
     pub authentication_type: Option<DBEnumWrapper<storage_enums::AuthenticationType>>,
     pub payment_method: Option<String>,
     pub payment_method_type: Option<String>,
+    pub client_source: Option<String>,
+    pub client_version: Option<String>,
```

Comment:
Do we need group by over client source & version?
wonder how we can enforce the cardinality of this parameter? If possible I'd prefer to only give filters over this and not client version

The other option is you can parse the client_source/version in the clickhouse MV to a select few options